docs/source/admin/log-messages.md

```diff
@@ -47,9 +44,9 @@ This log message is benign, and there is usually no action for you to take.
 ### Cause
 
 JupyterHub requires the `jupyterhub` python package installed inside the image or
-environment the user server starts in. This message indicates that the version of
+environment, the user server starts in. This message indicates that the version of
 the `jupyterhub` package installed inside the user image or environment is not
-the same version as the JupyterHub server itself. This is not necessarily always a
+the same as the JupyterHub server itself. This is not necessarily always a
```

Comment:
I know it's a bit repetitive, but I believe it's worth being as explicit as possible to eliminate all confusion.

```suggestion
the same as the JupyterHub server's version itself. This is not necessarily always a
```